src/libstd/build.rs

```diff
@@ -63,4 +63,8 @@ fn main() {
         println!("cargo:rustc-link-lib=c");
         println!("cargo:rustc-link-lib=compiler_rt");
     }
+    println!("cargo:rustc-env=STD_ENV_ARCH={}", env::var("CARGO_CFG_TARGET_ARCH").unwrap());
+    if target.contains("-none") || target.contains("nvptx") {
+        println!("cargo:rustc-cfg=feature=\"restricted-std\"");
```

Comment:
This seems like it'd be easy to forget to update this, would it be possible to invert this so there's a whitelist of targets to be stable? (or is that infeasible to create at this point?)

Reply:
To clarify, my worry is that a new target could be added and it'd accidentally forget to get added here to be unstable.

Reply:
Sure, that is definitely my preference.

Reply:
Nice this looks perfect 👍